openedx/core/djangoapps/user_api/accounts/views.py

```diff
@@ -806,7 +806,7 @@ def partial_update(self, request):
 
     def cleanup(self, request):
         """
-        DELETE /api/user/v1/accounts/update_retirement_status/
+        POST /api/user/v1/accounts/update_retirement_status/
```

Comment:
Nit: Would a `/delete_retirement_status` endpoint now be more appropriate?

Reply:
Bah it's right in the url, this is just copypasta that slipped through the original commit.